bark_ml/observers/graph_observer.py

```diff
@@ -20,23 +20,23 @@ class GraphObserver(StateObserver):
   
   def __init__(self,
                normalize_observations=True,
```

Comment:
If we use a parameter server, it might be useful to include normalize_observations too.